packages/cfpb-layout/src/molecules/card.less

```diff
@@ -111,4 +162,116 @@
             border-bottom-width: 3px;
         }
     }
+
+    // Topic cards.
+    &__topic {
+        text-align: center;
+        width: 170px;
+
+        & > a {
+            border: 1px solid @gray-20;
+            border-bottom-width: 3px;
+        }
+
+        .m-card_icon {
+            font-size: unit( 30px / @base-font-size-px, em );
+            color: @green;
+            margin-bottom: unit( 5px / @base-font-size-px, em );
+        }
+
+        &:hover {
+            > a {
+                box-shadow: 0 2px 0 0 inset @gray-20,
+                            2px 0 0 0 inset @gray-20,
+                           -2px 0 0 0 inset @gray-20;
+            }
+
+            .u-card-bottom-bar();
+        }
+        .u-card-link-text();
+    }
+
+    &__topic-action {
+        .m-card_icon {
+            color: @pacific;
+        }
+    }
+
+    // Breakout cards.
+    @card_img_width: 210px;
+    @card_img_height: 120px;
+    &__breakout {
+        min-width: 210px;
+        text-align: center;
+
+        // Provide padding offset set to half the image height.
+        > a {
+            padding-top: @card_img_height / 2;
+        }
+
+        .m-card_inner-wrapper {
+            position: relative;
+            z-index: 0;
+            min-height: @card_img_height + ( @grid_gutter-width / 2 );
+
+            background: @gray-5;
+            border: 1px solid @gray-20;
+            border-bottom-width: 3px;
+        }
+
+        .m-card_img {
+            position: absolute;
+            z-index: 1;
+            top: 0;
+            left: 50%;
+            margin-left: -105px;
+            width: @card_img_width;
+            height: @card_img_height;
+
+            // Match half of image width.
+            margin-left: -( @card_img_width / 2 );
+        }
+
+        .m-card_footer {
+            margin-top: ( @card_img_height / 2 ) + @grid_gutter-width;
+        }
+
+        &:hover {
+            .m-card_inner-wrapper {
+                box-shadow: 0 2px 0 0 inset @gray-20,
+                            2px 0 0 0 inset @gray-20,
+                           -2px 0 0 0 inset @gray-20;
+            }
+
+            .u-card-bottom-bar();
+        }
+        .u-card-link-text();
+    }
+
+    // Highlight cards.
+    &__highlight {
+        background: @green-20;
+
+        & > a {
+            border: 1px solid @green-40;
+            border-bottom-width: 3px;
+        }
+
+        h3,
+        p {
+            color: @black !important;
```

Comment:
Is `!important` necessary here? It seems to work fine without it.

Reply:
Hmmmmmmmm I guess not! I forget why I added it. Oh well, easy enough to add back if we need it. Removed in https://github.com/cfpb/design-system/pull/1277/commits/e5d3c2d496bec8154124ef8373d52ca0444dda0f